tap_github/schemas/commits.json

```diff
@@ -1,6 +1,5 @@
 {
   "type": ["null", "object"],
-  "additionalProperties": false,
```

Comment:
Any reason for removing it?

Reply:
It is already present and the End of the Doc, hence removed due to duplication